website/docs/cdktf/develop-custom-constructs/publishing-and-distribution.mdx

```diff
@@ -6,17 +6,17 @@ description: |-
 
 # Construct Publishing and Distribution
 
-Custom construct classes are reusable infrastructure configurations written in a programming language. After you [write a custom construct](/developing-custom-constructs/construct-design), you can translate it to multiple languages and publish it for others to import and use in CDK for Terraform (CDKTF) applications.
+Custom construct classes are reusable infrastructure configurations written in a programming language. After you [write a custom construct](https://terraform.io/developing-custom-constructs/construct-design), you can translate it to multiple languages and publish it for others to import and use in CDK for Terraform (CDKTF) applications.
```

Comment:
This link seems to be broken (and already was 😄) – I don't know what the link checker is going to catch, but this is definitely something our team should fix (either in this PR or in a separate one)

Reply:
I found 2 more! 😆 

My preference here would be for these links to be fixed outside of this PR. Otherwise, we'll see the tests for the files with broken links fail, because they'll be incorrect on the upstream deploy preview being tested as the source of truth.

If you'd like I can make that PR after we've got this one reviewed/approved/merged? 😊

Reply:
Ah, got it! Sure, no problem! A PR would be the best, but a list of broken links is fine, too.

Reply:
just saw your other comments 😄

Reply:
fixed here: https://github.com/hashicorp/terraform-cdk/pull/2693